doc/ref_model/chapters/chapter02.md

```diff
@@ -148,14 +148,15 @@ By trying to sort VNFs into different categories based on the requirements obser
     - Predictable computing
     - High network throughput
     - Low network latency
+<!--
```

Comment:
IMO, we shouldn't comment off Profile Three and Four. Profile 3 Requirements (Predictable computing and Low network latency) are a subset of Profile Two requirements. We do not want to write-off critical VNFs such as MME.

Reply:
i am not sure @pgoyal01 we didn't comment the workload, we only commented the profiles. added MME to the NI. hope that will cover your concern.

Reply:
fixed in: https://github.com/cntt-n/CNTT/pull/1064/commits/2057dda37901e2a62ca9f07b6eac82f215f51de2

Reply:
@rabi-abdel Below is the content that is commented out -- I have bolded the VNFs that need to be captured with the other profiles that need to be included in the remaining two profiles:

**Profile Three**
  - VNF types
    - Control plane functions with specific computing needs
    - _Examples:_ **MME, AMF, IMS-CSCF**
  - Requirements
    - Predictable computing
    - Low network latency    

- **Profile Four**
  - VNF types
    - Control plane functions with specific storage needs
    - _Example:_ **UDR** 
  - Requirements
    - High storage IOPS
    - High storage volume

Reply:
@pgoyal01 fixed.